addons/website/static/src/js/editor/snippets.editor.js

```diff
@@ -27,6 +32,35 @@ weSnippetEditor.SnippetsMenu.include({
         ['website-settings', _lt("Website Settings")],
     ],
 
+    /**
+     * @override
+     */
+    start() {
+        this._super(...arguments);
+        this.ownerDocument = this.$el[0].ownerDocument;
+        this.$currentAnimatedText = $();
+        const $animateTextButton = $(`
+            <div title="${_t('Animate text')}"
+                class="btn fa fa-play fa-fw o_we_animate_text">
+            </div>
+        `);
+        const $toolbar = this.options.wysiwyg.toolbar.$el;
+        $toolbar.find('.oe-toggle-checklist').after($animateTextButton).remove();
```

Comment:
ok, I extend the toolbar in the XML file for website. In the same way we extend it for website forum.
I hope I have met your request.

Reply:
Note that PDE asked me to keep the checklit button in the toolbar in massmailing

Reply:
@bvr-odoo 
massmailing? I don't think it's needed there, it might be a misunderstanding.
It will be important in task pad but that's different toolbar right?

Reply:
It's the same toolbar that is used everywhere ... what I meant is that I only removed the checklist button in the website editor.